### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by shashankkvs309668** [microsoft/playwright#24158]
Hi Iam currenlty using Playwright Test Framework with Node.js:
How can i run the below mentioned scripts with RunA & RunB from package.json which is having different cross-env plugin value. Currently the only way iam seeing is to run it differently and generate reports separate. But how can i run both the scripts at the same time and get all the data in one html report only.

### System info
- Playwright Version: [v1.35]
- Operating System: [All, Windows 11, Ubuntu 20, macOS 13.2, etc.]
- Browser: [All, Chromium, Firefox, WebKit]
- Other info:

### Source code
Scripts:{
Run1:'cross-env plugin=org.CompA npx playwright test feature-a.spec.js',
Run2:'cross-env plugin=org.CompB npx playwright test feature-b.spec.js' 
}

**Link to the GitHub repository with the repro**

[https://github.com/your_profile/playwright_issue_title]

**Comment by mxschmitt** [comment]
Looks like this can be folded into https://github.com/microsoft/playwright/issues/10437.

Another way of dealing would be to just create two projects and run them together without invoking them twice or [parameterising inline](https://playwright.dev/docs/test-parameterize), but this is most likely not what you want, since you want to have it applied to your whole suite.

**Comment by mxschmitt** [comment]
Folding into https://github.com/microsoft/playwright/issues/10437

We don't recommend running them in parallel manually, we recommend using projects instead and letting Playwright do the parallelization and merging.

**Comment by shashankkvs309668** [comment]
@mxschmitt  My question is now how can i run/handle two different cross-env plugins now i want to go in single run.Can you please help me in this

**Comment by mxschmitt** [comment]
Parameterised projects is for that: https://playwright.dev/docs/test-parameterize#parameterized-projects and your parameters could be env vars.

**Comment by shashankkvs309668** [comment]
@mxschmitt  Apologize for confusing all, My example is similar to the below one,
<Scripts>:
run1: cross-env PICKENVFILE=<file A> npx playwright featureSpec-a
run2: cross-env PICKENVFile=<file B> npx playwright featureSpec-b 

In This example im passing file A / file B  to pick all environment variables from the respective file using dot env internally at global-setup configuration , which is depending on spec file im choosing. In this case if i have to run both run1 & run2 how can i acheive it.
